q2_qemistree/_classyfire.py

```diff
@@ -78,14 +78,16 @@ def get_classyfire_taxonomy(feature_data: pd.DataFrame) -> pd.DataFrame:
             response = requests.get(to_classyfire+str(inchikey)+'.json')
             if response.status_code == 200:
                 response = response.json()
-                classyfire_levels_sub = [level for level in classyfire_levels
-                                         if level in response.keys()]
-                if len(classyfire_levels_sub) == 0:
+                sublevels = [level for level in classyfire_levels
+                             if level in response.keys()]
```

Comment:
the `.keys()` call is not necessary, and likely would be good to avoid as I believe that return is list-like and which is not great for search